.eslintrc.js

```diff
@@ -88,6 +88,8 @@ module.exports = {
     '@typescript-eslint/prefer-optional-chain': 0,
     '@typescript-eslint/prefer-nullish-coalescing': 0,
     '@typescript-eslint/no-misused-promises': 0,
+    '@typescript-eslint/no-unsafe-argument': 0,
```

Comment:
Why do we need to generically allow the usage of any? Seems like it would be better to disable this via in line comments than allow it by default.

Reply:
Ahh, I misunderstood the docs. Ok, yeah this makes more sense why it's being added then.